src/main/drivers/accgyro/accgyro_mpu.h

```diff
@@ -46,6 +46,7 @@
 #define ICM20649_WHO_AM_I_CONST             (0xE1)
 #define ICM20689_WHO_AM_I_CONST             (0x98)
 #define ICM42605_WHO_AM_I_CONST             (0x42)
+#define ICM42688P_WHO_AM_I_CONST           (0x47)
```

Comment:
:space_invader:

Reply:
what does that emoji mean?  whitespace?  whitespace not lining was probably as a result of a search/replace operation.  I'll fix that.